src/app/models/use-school-suggestion-list.ts

```diff
@@ -0,0 +1,75 @@
+import React, {useState, useMemo} from 'react';
```

Comment:
You might want to see the [before-prettier commit](https://github.com/openstax/os-webview/pull/2701/commits/52e871bc5da2652ccce454751e18e0945a8100c2#diff-6911a1c24657957d4e4c5e74fdbfc6ab869fff8ff6362cfe40de5d14e8ad7c15).